_pages/known-issues/hardware/trezor.md

```diff
@@ -8,6 +8,8 @@ title: Trezor
 Default usage leaks your xpub to Trezor's servers (and whoever they share records with).
 This is [estimated at 99% of their users](https://coldbit.com/how-to-use-a-trezor-without-leaking-your-balance-and-transaction-history-to-third-parties/).
 This has severe privacy implications and some [potential security implications](https://www.reddit.com/r/Bitcoin/comments/5csles/what_are_the_security_risks_of_sharing_xpub/).
+A simple solution is to initialize your Trezor with the Trezor Manager app (android) & use it with Specter/Core.
+The CLI solution, linked above, enables more options. 
```

Comment:
Can you clarify this? There's no mention of CLI solutions on the page.

Reply:
ya, that's not super clear...

the article by Coldbit, linked above, shows how to use Trezor in CLI... 

// i know this is a guide for multi-sig, but why not support best practices for single-sig users, while we're at it...

Reply:
I don't believe there are best practices for single sig, it's too hard for all but experts to do safely!